locations/spiders/apm_monaco.py

```
@@ -0,0 +1,29 @@
+
+class ApmMonacoSpider(JSONBlobSpider):
+    name = "apm_monaco"
+    item_attributes = {
+        "brand": "APM Monaco",
+        "brand_wikidata": "Q85738954",
+    }
+    start_urls = ["https://storemapper-herokuapp-com.global.ssl.fastly.net/api/users/12892/stores.js"]
+    locations_key = "stores"
+    custom_settings = {
+        "ROBOTSTXT_OBEY": False,
+    }
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)